app/views/GoldenTicketController/allMyGoldenTicketVotes.scala.html

```diff
@@ -13,21 +13,25 @@ <h1 class="panel-title"><i class="fas fa-chart-bar"></i> @Messages("gt.yourVotes
             <div class="card-body">
                 <div class="col-md-12">
                     <h3><i class="fas fa-flask"></i> @Messages("gt.personnal")</h3>
+                    @models.ConferenceDescriptor.ConferenceProposalTypes.ALL.filter(models.ConferenceDescriptor.ConferenceProposalConfigurations.accessibleTypeToGoldenTicketReviews(_)).map{confType=>
+                      <a href="@routes.GoldenTicketController.allMyGoldenTicketVotes(confType.id)" class="btn btn-sm @if(confType.id==talkType) { btn-success } else { btn-primary }">@Messages(confType.id)</a>
+                    }
+                    <br/>
+
                     @if(allMyVotes.nonEmpty){
-                        @Messages("gt.youReviewed", allMyVotes.count(_._2.isDefined))<br>
+                        @Messages("gt.youReviewed", proposalsNotReviewedCount + allMyVotes.size, allMyVotes.size, proposalsNotReviewedCount)
+                        @if(firstProposalNotReviewed.isDefined){
+                            <a href="@routes.GoldenTicketController.openForReview(firstProposalNotReviewed.get.id)" title="Open for review" class="btn btn-primary btn-sm">@Messages("admin.btn.review")</a>
+                        }
+                        <br>
 
-                        @defining(allMyVotes.filter(_._2.isDefined).filterNot(_._2 == Some(0)).toList.map(_._2.get)){allScores=>
-                            @Messages("gt.youVoted", allMyVotes.count(s => s._2.isDefined && s._2 != Some(0)))<br>
+                        @defining(allMyVotesExcludingAbst){allScores=>
```

Comment:
```suggestion
                        @defining(allMyVotesExcludingAbst.map(_._2)){allScores=>
```

It's not compiling otherwise :

```
Error:(51, 102) type mismatch;
 found   : List[(String, Double)]
 required: List[Double]
                            """),_display_(/*30.30*/Messages("gt.averageVote", library.Stats.average(allScores))),format.raw/*30.90*/("""
```

It is the change you did in the CFP side

Reply:
I really need to setup my dev environment correctly... at least to avoid such compilation failures

Regarding DB : this evening, I will try to setup again the dev environment with an empty Redis (had a look at the `README` and it seems like I should be able to start something from scratch)
Just crossing fingers about the fact that I don't need data populated into the Redis in order to make it work :-)

Reply:
OK I setup my dev environment and fixed/pushed the compilation fixes
![All_my_votes_-_Devoxx_France_2020](https://user-images.githubusercontent.com/603815/71674840-a3788280-2d7c-11ea-982b-b20e670e9dfb.png)
![All_my_votes_-_Devoxx_France_2020](https://user-images.githubusercontent.com/603815/71674858-ad9a8100-2d7c-11ea-8dd9-3685f5f306b7.png)


I think it will be easier for me now (I was completely blind until now)